retis/src/core/events/bpf.rs

```diff
@@ -78,7 +78,7 @@ macro_rules! raw_to_string_opt {
 /// The return value of EventFactory::next_event()
 pub(crate) enum EventResult {
     /// The Factory was able to create a new event.
-    Event(Event),
+    Event(Box<Event>),
```

Comment:
What is the reason behind setting the event in a box?

Reply:
Oh, now I remember hehe took some time:
https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant

Reply:
Interesting. I'm wondering if there is a gain here as the other variant is not used widely. Anyway that's fine.